bytecomp/translmod.ml

```diff
@@ -686,9 +688,11 @@ and transl_structure loc fields cc rootpath final_env = function
                     transl_module Tcoerce_none None od.open_expr, body), size
           end
       | Tstr_modtype _
-      | Tstr_class_type _
-      | Tstr_attribute _ ->
+      | Tstr_class_type _ ->
           transl_structure loc fields cc rootpath final_env rem
+      | Tstr_attribute attr ->
+          Builtin_attributes.warning_scope ~ppwarning:false [attr]
+            (fun () -> transl_structure loc fields cc rootpath final_env rem)
```

Comment:
It would be better to useBuiltin_attributes.warning_attribute here (in line with what Typemod does).